src/main/java/utils/Convertor.java

```diff
@@ -0,0 +1,13 @@
+package utils;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Convertor {
+
+    public static List<String> convertStringToList(String input) {
+        String[] names = input.split(",");
+
+        return new ArrayList<>(List.of(names));
```

Comment:
```suggestion
        String[] names = input.split(",");

        return List.of(names);
```
new ArrayList<>() 부분은 삭제해도 동일하게 작동해요!